components/MDX/UpcomingEvents.tsx

```diff
@@ -0,0 +1,36 @@
+import { getTranslations } from 'next-intl/server';
```

Comment:
If there's no upcoming event, I think displaying just one BlogCard would be better than two, which would be repetitive. We could, for example, center it in the middle of the page.

Reply:
Displaying always two signals that there's always two being displayed. It's a design pattern used by many places.